SimPEG/EM/Static/SIP/RxSIP.py

```diff
@@ -53,9 +58,15 @@ def evalDeriv(self, src, mesh, f, v, adjoint=False):
 # DC.Rx.Dipole(locs)
 class Dipole(BaseRx):
 
+    rxgeom = "dipole"
```

Comment:
what does this refer to? if it is switching between pole / dipole, can we treat those as separate classes, similar to how the dc problem is done?